From 671e8e03fb9a975c7a8513a190d5195d2442318e Mon Sep 17 00:00:00 2001
From: Alexandre TORGUE <alexandre.torgue@st.com>
Date: Wed, 2 Oct 2024 16:26:23 +0200
Subject: [PATCH 731/841] feat(fdts): update stm32mp215f-dk system
 configuration files

- Replace hard coded '0' value by 'RIF_UNUSED' in fw-config file.
- Fix typo Mhz/MHz in rcc file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git e7759b83949795f7b8a04c744a03560ed684ee75 feat(fdts): update stm32mp215f-dk system configuration files]
Change-Id: I2f7b7e8c1e6a6cbd010e8fa30ec6c17ff63496b7
Signed-off-by: Alexandre TORGUE <alexandre.torgue@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/408557
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Maxime MERE <maxime.mere@foss.st.com>
---
 fdts/stm32mp215f-dk-ca35tdcid-fw-config.dtsi |  6 +++---
 fdts/stm32mp215f-dk-ca35tdcid-rcc.dtsi       | 15 +++++++--------
 2 files changed, 10 insertions(+), 11 deletions(-)

diff --git a/fdts/stm32mp215f-dk-ca35tdcid-fw-config.dtsi b/fdts/stm32mp215f-dk-ca35tdcid-fw-config.dtsi
index 78aeea0aa..813bf36a4 100644
--- a/fdts/stm32mp215f-dk-ca35tdcid-fw-config.dtsi
+++ b/fdts/stm32mp215f-dk-ca35tdcid-fw-config.dtsi
@@ -6,7 +6,7 @@
 /*
  * stm32mp215f tf-a firmware config
  * Project : open
- * Generated by XLmx tool version 2.2 - 6/4/2024 2:38:32 PM
+ * Generated by XLmx tool version 2.2 - 10/2/2024 3:58:37 PM
  */
 
 / {
@@ -24,11 +24,11 @@
 	st-mem-firewall {
 		bl31_context: bl31-context@81fc0000 {
 			reg = <0x0 0x81fc0000 0x0 0x40000>;
-			st,protreg = <RISAFPROT(RISAF_REG_ID(7), RIF_CID0_BF|RIF_CID1_BF, RIF_CID0_BF|RIF_CID1_BF, 0, RIF_SEC, RIF_ENC_EN, RIF_BREN_EN)>;
+			st,protreg = <RISAFPROT(RISAF_REG_ID(7), RIF_CID0_BF|RIF_CID1_BF, RIF_CID0_BF|RIF_CID1_BF, RIF_UNUSED, RIF_SEC, RIF_ENC_EN, RIF_BREN_EN)>;
 		};
 		op_tee: op-tee@82000000 {
 			reg = <0x0 0x82000000 0x0 0x2000000>;
-			st,protreg = <RISAFPROT(RISAF_REG_ID(8), RIF_CID0_BF|RIF_CID1_BF, RIF_CID0_BF|RIF_CID1_BF, 0, RIF_SEC, RIF_ENC_EN, RIF_BREN_EN)>;
+			st,protreg = <RISAFPROT(RISAF_REG_ID(8), RIF_CID0_BF|RIF_CID1_BF, RIF_CID0_BF|RIF_CID1_BF, RIF_UNUSED, RIF_SEC, RIF_ENC_EN, RIF_BREN_EN)>;
 		};
 	};
 };
diff --git a/fdts/stm32mp215f-dk-ca35tdcid-rcc.dtsi b/fdts/stm32mp215f-dk-ca35tdcid-rcc.dtsi
index a12ad264d..d558c3438 100644
--- a/fdts/stm32mp215f-dk-ca35tdcid-rcc.dtsi
+++ b/fdts/stm32mp215f-dk-ca35tdcid-rcc.dtsi
@@ -7,7 +7,7 @@
 /*
  * stm32mp215f Clock tree device tree configuration
  * Project : open
- * Generated by XLmx tool version 2.2 - 6/4/2024 2:38:31 PM
+ * Generated by XLmx tool version 2.2 - 10/2/2024 3:58:35 PM
  */
 
 &clk_hse {
@@ -60,30 +60,29 @@
 	>;
 
 	pll1: st,pll-1 {
-		st,pll = <&pll1_cfg_1200Mhz>;
+		st,pll = <&pll1_cfg_1200MHz>;
 
-		pll1_cfg_1200Mhz: pll1-cfg-1200Mhz {
+		pll1_cfg_1200MHz: pll1-cfg-1200MHz {
 			cfg = <30 1 1 1>;
 			src = <MUX_CFG(MUX_MUXSEL5, MUXSEL_HSE)>;
 		};
 	};
 
 	pll2: st,pll-2 {
-		st,pll = <&pll2_cfg_400Mhz>;
+		st,pll = <&pll2_cfg_400MHz>;
 
-		pll2_cfg_400Mhz: pll2-cfg-400Mhz {
+		pll2_cfg_400MHz: pll2-cfg-400MHz {
 			cfg = <20 1 1 2>;
 			src = <MUX_CFG(MUX_MUXSEL6, MUXSEL_HSE)>;
 		};
 	};
 
 	pll4: st,pll-4 {
-		st,pll = <&pll4_cfg_1200Mhz>;
+		st,pll = <&pll4_cfg_1200MHz>;
 
-		pll4_cfg_1200Mhz: pll4-cfg-1200Mhz {
+		pll4_cfg_1200MHz: pll4-cfg-1200MHz {
 			cfg = <30 1 1 1>;
 			src = <MUX_CFG(MUX_MUXSEL0, MUXSEL_HSE)>;
 		};
 	};
 };
-
-- 
2.47.2

